.github/workflows/test.yml

```diff
@@ -34,4 +34,5 @@ jobs:
       target_path: 'rust-bitvmx-client/target'
       cargo_features: 'testpanic'
       nightly: false
-    secrets: inherit
\ No newline at end of file
+      submodule_branches: '{"rust-bitvmx-key-manager": "dev", "rust-bitvmx-protocol-builder": "keymanager-adapt", "rust-bitvmx-wallet": "keymanager-adapt", "rust-bitvmx-storage-backend": "v0.1.1-review.1"}'
```

Comment:
we aware before merging, this was added for test to pass using the right version, i don't think we want to merge this